lib/GADS/Datum/Rag.pm

```diff
@@ -66,6 +68,19 @@ sub write_value
     $self->write_cache('ragval');
 }
 
+sub as_grade
+{
+    my $self = shift;
+    my %mapping = (
+        a_grey   => 'undefined',
+        b_red    => 'negative',
+        c_amber  => 'average',
+        d_green  => 'positive',
+        e_purple => 'unexpected'
```

Comment:
I think the Bootstrap definitions are better (warning, danger etc). For example, amber is normally used as some sort of warning.